lidarSuit/dataOperator.py

```diff
@@ -420,6 +428,14 @@ def dataTransform90(self):
 
         return self
 
+    def vertical_component_check(self, check90):
+
+        if check90:
+            if not hasattr(self.data, "radial_wind_speed90"):
+                raise KeyError
+        else:
+            print("Vertical component check was ignored.")
```

Comment:
I tend to use log instead of print, so it is easier to control how verbose I want it. But it's your call.

Reply:
I like your idea, and I will use it in the future.